src/main/java/com/greenfox/lvlup/service/BadgeLevelService.java

```diff
@@ -0,0 +1,40 @@
+package com.greenfox.lvlup.service;
+
+import com.greenfox.lvlup.model.entity.Badge;
+import com.greenfox.lvlup.model.entity.BadgeLevel;
+import com.greenfox.lvlup.model.entity.User;
+import com.greenfox.lvlup.model.entityTestingDto.BadgeDTO;
+import com.greenfox.lvlup.model.entityTestingDto.BadgeLevelDTO;
+import com.greenfox.lvlup.repositrory.BadgeLevelRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Service
+public class BadgeLevelService {
+    private UserService userService;
+
+    @Autowired
+    public BadgeLevelService(UserService userService) {
+        this.userService = userService;
+    }
+
+    public BadgeLevelDTO getDTOfromBadgeLevel(BadgeLevel badgeLevel) {
```

Comment:
just a suggestion for naming such methods: maybe a format like 'convertWhateverEntityToDTO" could be more descriptive?

Reply:
there's a couple more method names like this further down, and I got confused by the 'get' part at one point, that's why I'm asking